framework/test/functional/queue.test.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const Queue = require('../../src/queue');
+
+describe('Test queue', () => {
+	let queue;
+	const redisEndpoint = 'redis://localhost:6379/0';
```

Comment:
I think that should be configurable, depending on the CI server:
```suggestion
	const redisEndpoint = process.env.REDIS_URL || 'redis://localhost:6379/0';
```